src/style.css

```diff
@@ -1 +1,112 @@
-/* add styles here */
+html {
+  --main-color: var(--main-color);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url('fonts/Avenir.ttc');
+}
+
+@font-face {
+  font-family: Avenir-Book;
+  src: url('fonts/Avenir-Book.ttf');
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url('fonts/Avenir-Heavy.ttf');
+}
+
+body {
+  box-sizing: border-box;
+  color: var(--main-color);
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+}
+
+.search-bar__input {
+  width: 100%;
+}
+
+.search-bar__input--big {
+  height: 70px;
+  padding-left: 62px;
+
+  box-sizing: border-box;
+  background-color: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  outline: none;
+
+  /* Text */
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+  line-height: 22px;
+  color: var(--main-color);
+}
+
+.search-bar__input:hover {
+  box-shadow: 0 3px 8px 0 rgba(61, 78, 97, 0.2);
+}
+
+.search-bar__input:focus {
+  font-family: Avenir-Heavy, sans-serif;
+  color: var(--main-color);
+  outline: 0;
+  border-radius: 3px;
+  border: 1px solid #e1e7ed;
+
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  box-shadow: 0 2px 2px 0 rgba(0, 0, 0, 0.25);
+  text-shadow: 0 4px 4px 0 rgba(0, 0, 0, 0.25);
+}
+
+.search-bar__input--small {
+  height: 42px;
+  padding-left: 33px;
+
+  font-family: Avenir, sans-serif;
+  font-size: 14px;
+  line-height: 19px;
+  letter-spacing: 0;
+  text-shadow: none;
+
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  outline: none;
+}
+
+.search-bar__label {
+  margin-top: 20px;
+  display: block;
+  position: relative;
+}
+
+.search-bar__label--big::before {
```

Comment:
move common styles for `.search-bar__label--big::before and .search-bar__label--small::before`  to `.search-bar__label::before` and then add specific styles separately for both of them

Reply:
try to reuse common styles and don't repeat yourself(when it's possible and make sense)

Reply:
Moved common styles for ```.search-bar__label--big::before``` and ```.search-bar__label--small::before to .search-bar__label::before```.